docs/contributing.md

```diff
@@ -40,6 +40,8 @@ provided `creds.env.example` file to `creds.env` and customizing the contents of
 
 ### Testing
 
+While the plugin itself supports any general release version of Nautobot, the testing requires at minimum 1.0.2.
```

Comment:
Just double-checking - is "1.0.2" correct here or did you mean "1.0.3"? If the former, what's different in 1.0.1 that it would not support the tests?

Reply:
Well... Tests will fail if you use 1.0.0 or 1.0.1.  They will pass with 1.0.2, but the choices test won't run until 1.0.3.  So should I change minimum to 1.0.3?

Reply:
1.0.2 sounds fine/correct to me then. Thanks! :)